website/src/logged_in/logs/Logs.js

```diff
@@ -76,6 +76,147 @@ const columns = [
             }
         }
     },
+    {
+        field: 'minF&B',
```

Comment:
What is this field? Where in the data model does this field exist?

Note: fields presented to users do ***NOT*** correspond to what the model uses.

Reply:
I found a document named "Pricing modeling data" that the DPA team uses to train their model so I did a first draft based on the data type showed on that excel form. I asked the DPA team for verification that what data we actually want to present on this page. After getting their reply, I will update the field.

Reply:
You don't need to ask the DPA team. You can see on the website which parameters the user enters into the tool. We want to display those, not the internal parameters that the DPA team uses to compute the outputs.